models/Translation.php

```diff
@@ -0,0 +1,478 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @category   Pimcore
+ * @package    Translation
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Model;
+
+use Pimcore\Cache;
+use Pimcore\Cache\Runtime;
+use Pimcore\Event\Model\TranslationEvent;
+use Pimcore\Event\TranslationEvents;
+use Pimcore\File;
+use Pimcore\Model\Translation\TranslationInterface;
+use Pimcore\Model\Translation\Website;
+use Pimcore\Tool;
+
+/**
+ * @method \Pimcore\Model\Translation\Dao getDao()
+ */
+class Translation extends Website implements TranslationInterface
```

Comment:
Maybe we should just rename `models/Translation/AbstractTranslation.php` to `models/Translation/Translation.php` (and the class name) to keep the Git history and then do the following for the `AbstractTranslation`

```php
abstract class AbstractTranslation extends Translation
```

This would keep things comaptible and we're not duplicating code.

Reply:
This would also make #7808 easier, as we're just deleting the deprecated classesand while keeping the history of the `AbstractTranslation` in `Translation`.